src/GitHub.VisualStudio/source.extension.vsixmanifest

```diff
@@ -10,7 +10,7 @@
     <PreviewImage>Resources\preview_200x200.png</PreviewImage>
     <Tags>GitHub git github.com team explorer</Tags>
   </Metadata>
-  <Installation AllUsers="true">
+  <Installation>
```

Comment:
This change should never be commited, change it back before you're done with this branch.